### DISCUSSION TITLE
Maximum number of nodes?

### DISCUSSION THREAD

**Discussion by Getsumi3** [graphia-app/graphia#32] [Q&A] [answered]
Hello. 
This tool looks interesting but before using it I'd like to know what is (aproximatively) maximum number of nodes that the user can visualize?

Thanks.

**Comment by timangus** [comment, marked as answer]
It's obviously hardware dependent, but at the top end you're talking single digit millions of nodes/edges, albeit not very smoothly.

**Comment by Getsumi3** [comment]
I see. Thanks.
Right now I'm looking to render somewhere around 20-50 millions of nodes

**Comment by timangus** [comment]
I'm not aware of any software that will do that interactively, unfortunately. You can of course use a variety of languages/packages/frameworks etc. to render static images; this is probably your best option for graphs of that scale.
